manager/container.go

```diff
@@ -43,7 +43,7 @@ import (
 // Housekeeping interval.
 var HousekeepingInterval = flag.Duration("housekeeping_interval", 1*time.Second, "Interval between container housekeepings")
 
-var cgroupPathRegExp = regexp.MustCompile(`.*devices.*:(.*?)[,;$].*`)
+var cgroupPathRegExp = regexp.MustCompile(`.*devices.*?:(.*?)[,;$].*`)
```

Comment:
I think using a negated character set is prefered to lazy matching. Also, the `.*` on either end isn't doing anything except slowing this down.